src/render/painter.js

```diff
@@ -837,15 +836,13 @@ class Painter {
     /**
      * Checks whether a pattern image is needed, and if it is, whether it is not loaded.
      *
-* @returns true if a needed image is missing and rendering needs to be skipped.
+     * @returns true if a needed image is missing and rendering needs to be skipped.
      * @private
      */
-    isPatternMissing(image: ?CrossFaded<ResolvedImage>): boolean {
-        if (!image) return false;
-        if (!image.from || !image.to) return true;
-        const imagePosA = this.imageManager.getPattern(image.from.toString());
-        const imagePosB = this.imageManager.getPattern(image.to.toString());
-        return !imagePosA || !imagePosB;
+    isPatternMissing(image: ?ResolvedImage): boolean {
+        if (image === null) return true;
+        if (image === undefined) return false;
```

Comment:
Yes, now instead of `from` and `to` being `null` when it's unset, you have `image === null`, so I had to change it this way for the logic to remain the same (there was a failing test otherwise).

Reply:
Ok! Thanks for confirming.